client/filter/table_filter_list.go

```diff
@@ -0,0 +1,102 @@
+/*-
+ * #%L
+ * OBKV Table Client Framework
+ * %%
+ * Copyright (C) 2021 OceanBase
+ * %%
+ * OBKV Table Client Framework is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ * See the Mulan PSL v2 for more details.
+ * #L%
+ */
+
+package filter
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type Operator uint8
+
+const (
+	OperatorAnd Operator = iota
+	OperatorOr
+)
+
+type ObTableFilterList struct {
+	op      Operator
+	filters []ObTableFilter
+}
+
+func NewObTableFilterList() *ObTableFilterList {
```

Comment:
I guess we do not need ``NewObTableFilterList()``, since there are two other construct functions.
On the other hand, there is no interface for users to change Operator, it is easy to make mistakes using this function.

Reply:
agree.

Reply:
fixed.